docs/design-docs/08-package-variant.md

```diff
@@ -1183,16 +1226,16 @@ them. That is, there are certain objects that are accessible within the CEL
 expression. For CEL expressions used in the PackageVariantSet `template` field,
 the following variables are available:
 
-| CEL Variable | Variable Contents                                            |
-| -------------| ------------------------------------------------------------ |
-| repo         | The default repository name based on the targeting criteria. |
-| package      | The default package name based on the targeting criteria.    |
-| upstream     | The upstream PackageRevision.                                |
-| repository   | The downstream Repository.                                   |
+| CEL Variable   | Variable Contents                                            |
+| -------------- | ------------------------------------------------------------ |
+| repoDefault    | The default repository name based on the targeting criteria. |
+| packageDefault | The default package name based on the targeting criteria.    |
+| upstream       | The upstream PackageRevision.                                |
+| repository     | The downstream Repository.                                   |
```

Comment:
Add `target` here too?

Reply:
It's described in the paragraph below, because it's contented varies based on the type of target. But I will add it here in a follow up PR.